docs/docs/docs/01-core/admin/03-settings/01-review-workflows.mdx

```diff
@@ -40,6 +82,7 @@ The API returns an `array` of workflows.
 
 ```ts
 useReviewWorkflows(queryParams: object): {
+  meta: { workflowCount: int }
```

Comment:
im gonna be annoying here, `int` is not a type, `number` is 😄

Reply:
In python it is! :p Thanks - will fix.